src/api.js

```diff
@@ -0,0 +1,86 @@
+let debug = require('debug')('app:main');
+let APIBuilder = require('taskcluster-lib-api');
+let taskcluster = require('taskcluster-client');
+let uuid = require('uuid');
+
+let builder = new APIBuilder({
+  title:    'AMQP Messages API Documentation',
+  description: [
+    'This service is responsible for making pulse messages accessible',
+    'from browsers and cli. There are API endpoints to',
+    'bind / unbind to an exchange and pause / resume listening from a queue',
+  ].join('\n'),
+  serviceName: 'tcevents',
+  version: 'v1',
+  errorCodes: {
+    InputValidation:  400, // For JSON schema errors (bad exchange).
+  },
+  context: ['connection'],
+});
+
+builder.declare({
+  method: 'get',
+  route: '/connect/',
+  name: 'connect',
+  description: 'Connect to receive messages',
+  stability: APIBuilder.stability.experimental,
+  // Add input validation yml
+  title: 'Events-Api',
+}, async function(req, res) {
+  console.log("hello");
```

Comment:
Use `debug` for these statements... and then set `DEBUG='*'` when running your service... example `DEBUG='events:*' yarn test`.

Note: you're currently doing `require('debug')('app:main')`, but it would probably make more sense to do `require('debug')('events:api')`, or `require('debug')('app:api')`